controls/radrichtextbox/features/document-ruler.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Document Ruler
+page_title: Document Ruler
+description: Check our &quot;Document Ruler&quot; documentation article for the RadRichTextBox control.
+slug: radrichtextbox-features-document-ruler
+tags: document ruler, ruler, richtextbox, document editor, editor
+published: True
+---
+
+# Document Ruler
+
+The `DocumentRuler` element allows you to visualize a guide that will assist you in aligning text, tables, graphics, and other elements. To display the document ruler, wrap it around a `RadRichTextBox` instance.
```

Comment:
assist you in aligning text, tables, graphics, and other elements - How? Give an example. Add links to the TabStop and Section Column articles as well